charts/prefect-worker/README.md

```diff
@@ -241,17 +241,50 @@ prefect work-pool get-default-base-job-template --type kubernetes > base-job-tem
 helm install prefect-worker prefect/prefect-worker -f values.yaml --set-file worker.config.baseJobTemplate.configuration=base-job-template.json
 ```
 
+#### Using the Base Job Template
+
+The worker uses the [base job template](https://docs.prefect.io/v3/deploy/infrastructure-concepts/work-pools#base-job-template)
+to create the Kubernetes job that executes your workflow. The base job template configuration can be modified by setting
+`worker.config.baseJobTemplate.configuration`. For example, to provide image pull secrets for the container running your flow,
+provide the following values:
+
+```yaml
+worker:
+  config:
+    baseJobTemplate:
+      configuration: |
```

Comment:
quick question -- if you provide this and nothing else, does this merge with the base base job template? or will it overwrite it to _just_ contain this information?

Reply:
Great question - I checked the [source code](https://github.com/PrefectHQ/prefect/blob/e38b549fa2ae4cd70c5a18bb0dc9cb34cf2ca1fa/src/prefect/cli/work_pool.py#L414-L415) and it looks like we don't merge it over the default, we just take whatever is given (using `prefect work-pool update`). Looks like this is a little different than providing overrides in `prefect.yaml`, for example.

As a test, I created a work pool with the default base job template added under `worker.config.baseJobTemplate`. Then I removed a field, in this case `restartPolicy`, and upgraded the chart. At that point, `restartPolicy` was no longer present. This confirms there's no merging with the base. I'll add a note 👍🏼